GSA/digitalgov.gov
# New resource-topics landing page

## Summary
Redesign for the the [resources landing page](https://digital.gov/resources/) to be organized by topics.

### Preview

[Link to Preview](https://federalist-466b7d92-5da1-4208-974f-d61fd4348571.sites.pages.cloud.gov/preview/gsa/digitalgov.gov/nl-update-topics-page/resources/)

<details><summary>Screenshot</summary>
<p>

![Guides and Resources _ Digital_gov · 11 11am · 05-17](https://github.com/GSA/digitalgov.gov/assets/104778659/32d5c3f0-2953-4d13-93da-0c4e6c711bbe)

</p>
</details> 



### Solution

### How To Test

1. Visit /resources and see new page layout
2. Should see the following resource categories (left to right)

| category |
|--------|
| Design |
| Data & analysis |
| Operations |
| Content & communication |
| Technology | 
| Strategic development | 

3. Topics are in sentence case and listed in alphabetical order
4. Topic links work as expected

---

### Dev Checklist
- [x] Submit https://digital.gov/resources/ to wayback machine
